renderer/plugin.js

```diff
@@ -198,6 +201,14 @@ org.ekstep.contentrenderer.questionUnitPlugin = Plugin.extend({
    */
   getAudioIcon:function(path){
     return this.getAssetUrl(org.ekstep.pluginframework.pluginManager.resolvePluginResource(this._manifest.id, this._manifest.ver, path));
+  },
+  renderMath: function (event) {
+    setTimeout(function () {
```

Comment:
We need to test this in Low end devices to verify the set timeout will not cause any issue.